x/interchainqueries/client/cli/query.go

```diff
@@ -112,3 +113,27 @@ func CmdQueryRegisteredQueryResult() *cobra.Command {
 
 	return cmd
 }
+
+func CmdQueryLastRemoteHeight() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "query-last-remote-height [connection-id]",
+		Short: "queries last remote height by connection id",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx := client.GetClientContextFromCmd(cmd)
+			connectionId := args[0]
+			queryClient := types.NewQueryClient(clientCtx)
+			println("connection id", connectionId)
```

Comment:
Seems excessive

Reply:
woops, that looks like some post-debugging remnant

Reply:
lol y don't I see the comment in the Changes section? otherwise I wouldn't have left the same one

Reply:
imo GitHub review UI is very underdeveloped